back/sssscs/src/main/java/com/ib/config/ServerConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.ib.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class ServerConfig implements WebMvcConfigurer {
```

Comment:
Remove EnableWebMvc here too. Will certainly cause issues. Eg DateTime serialization works better without it

Reply:
Fixed in [526576c](https://github.com/magley/sssscs/pull/1/commits/526576cf20344a23c92be7c7ecad2359058da605).